tonic/src/transport/server/mod.rs

```diff
@@ -592,6 +592,7 @@ impl Service<Request<Body>> for Unimplemented {
             http::Response::builder()
                 .status(200)
                 .header("grpc-status", "12")
+                .header("content-type", "application/grpc+proto")
```

Comment:
```suggestion
                .header("content-type", "application/grpc")
```

The spec is actually incorrect :( so we need to remove that part, otherwise this is great!